.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
+# Install dependencies dependencies
+before_install:
+    - sudo apt-get install -y librrd-dev
+# Install dependencies
+install:
+    - pip install -r requirements.txt
+    - pip install -r test-requirements.txt
+# Run unit tests
+script:
+    - python -m unittest discover -s export/ -p '*_test.py'
+    - python -m unittest discover -s monitoring/ -p '*_test.py'
+    - python -m unittest discover -s plugin/ -p '*_test.py'
+    - python -m unittest discover -s system/vsys/ -p '*_test.py'
```

Comment:
I'm getting this error. I'm investigating, but I'm not sure how to fix it yet.

The command "python -m unittest2 discover -s export/ -p '*_test.py'" exited with 1.
0.01s$ python -m unittest2 discover -s monitoring/ -p '*_test.py'
/home/travis/virtualenv/python2.6.9/bin/python: unittest2 is a package and cannot be directly executed